tests/integration/test_get_analyses.py

```diff
@@ -0,0 +1,63 @@
+from http import HTTPStatus
+from flask.testing import FlaskClient
+from trailblazer.constants import TrailblazerStatus
+
+from trailblazer.store.models import Analysis
+
+
+def test_get_analyses(client: FlaskClient, analyses: list[Analysis]):
+    # GIVEN some analyses
+
+    # WHEN retrieving all analyses
+    response = client.get("/api/v1/analyses")
+
+    # THEN it gives a success response
+    assert response.status_code == HTTPStatus.OK
+
+    # THEN it should return all analyses
+    assert len(response.json["analyses"]) == len(analyses)
```

Comment:
should this test for the content too? you could add an assertion checking that all elements of the iterable are of type `Analysis`. Feel free to resolve if this is not applicable